common/scala/build.gradle

```diff
@@ -64,6 +64,14 @@ dependencies {
     compile 'io.kamon:kamon-statsd_2.11:0.6.7'
     //for mesos
     compile 'com.adobe.api.platform.runtime:mesos-actor:0.0.7'
+
+    //tracing support
+    compile group:'io.opentracing', name:'opentracing-api', version: '0.31.0'
+    compile group:'io.opentracing', name:'opentracing-util', version: '0.31.0'
+    compile group:'io.opentracing.brave', name:'brave-opentracing', version: '0.31.0'
+    compile group:'io.zipkin.reporter2', name:'zipkin-sender-okhttp3', version: '2.6.1'
+    compile group:'io.zipkin.reporter2', name:'zipkin-reporter', version: '2.6.1'
```

Comment:
Can we change this to use the same form as we use everywhere (or adjust the form everywhere to this scheme) for good measure?